disperser/dataapi/v2/server_v2.go

```diff
@@ -128,6 +128,10 @@ type (
 		StakePercentage         float64 `json:"stake_percentage"`
 	}
 	OperatorsSigningInfoResponse struct {
+		StartBlock          uint32                 `json:"start_block"`
+		EndBlock            uint32                 `json:"end_block"`
+		StartTimeUnixSec    int64                  `json:"start_time_unix_sec"`
```

Comment:
I want to add block range that the signing rates were computed from, but add the timestamp range as well for convenience cc @pschork since you asked